tests/integration/records/views/test_views_data.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+def test_data_application_json_get(api_client, db, create_record_factory):
+    record = create_record_factory("dat", with_indexing=True)
+    record_control_number = record.json["control_number"]
+
+    expected_status_code = 200
+    response = api_client.get(f"/data/{record_control_number}")
+    response_status_code = response.status_code
+
+    assert expected_status_code == response_status_code
+
+
+def test_data_application_json_put(api_client, db, create_record_factory):
+    record = create_record_factory("dat", with_indexing=True)
+    record_control_number = record.json["control_number"]
+
+    expected_status_code = 401
+    response = api_client.put(f"/data/{record_control_number}")
+    response_status_code = response.status_code
+
+    assert expected_status_code == response_status_code
+
+
+def test_data_application_json_delete(api_client, db, create_record_factory):
+    record = create_record_factory("dat", with_indexing=True)
+    record_control_number = record.json["control_number"]
+
+    expected_status_code = 401
+    response = api_client.delete(f"/data/{record_control_number}")
+    response_status_code = response.status_code
+
+    assert expected_status_code == response_status_code
+
+
+def test_data_application_json_post(api_client, db):
+    expected_status_code = 401
+    response = api_client.post("/data")
+    response_status_code = response.status_code
+
+    assert expected_status_code == response_status_code
+
+
+def test_data_search_json_get(api_client, db, create_record_factory):
+    create_record_factory("dat", with_indexing=True)
+
+    expected_status_code = 200
+    response = api_client.get("/data")
+    response_status_code = response.status_code
+
+    assert expected_status_code == response_status_code
+
+
+def test_data_record_search_results(api_client, db, es_clear, create_record):
```

Comment:
isn't this the same as above? because we test also the expected? (we can delete`` test_data_search_json_get``) this applies on the others too